assets/package-lock.json

```diff
@@ -20,22 +20,14 @@
       },
       "devDependencies": {
         "bulma": "^0.9.4",
-        "esbuild": "^0.17.7",
+        "esbuild": "^0.19.4",
         "esbuild-sass-plugin": "^2.9.0",
         "sass": "^1.63.3"
       }
     },
-    "../deps/phoenix": {
-      "version": "1.6.16",
-      "license": "MIT"
-    },
-    "../deps/phoenix_html": {
-      "version": "3.3.1"
-    },
-    "../deps/phoenix_live_view": {
-      "version": "0.17.14",
-      "license": "MIT"
-    },
+    "../deps/phoenix": {},
```

Comment:
Why do you omit the version and license tags?

Reply:
I imagine this might be because we have gone from lockfileVersion 2 to lockfileVersion 3.